src/delete_old_sessions.py

```diff
@@ -0,0 +1,40 @@
+from flask_sqlalchemy import SQLAlchemy
+from app import app, db
+from app.coursegrab.models.session import Session
+import datetime
+
+""" Deletes user sessions that have been last used 10 weeks or more ago """
+def delete_old_sessions():
+  sessions = Session.query.all()
+  outdated_sessions = []
+  for s in sessions:
+    if s.session_expiration <= datetime.datetime.now() or s.last_used <= (datetime.datetime.now() - datetime.timedelta(weeks=10)): 
```

Comment:
I dont think the part about `session_expiration` should be included (lmk if I'm wrong lol)
* `session_expiration` and `last_used` are different in nature that:
  * `session_expiration` is the expiration time of the token
  * `last_used` is when the user was last active with the corresponding `Session` object
* let's say there is `Session1` where token is expired (session_exp). But the user can still use `Session1` by sending in a `refresh_token` and backend will assign a new `token`
* so there could be an instance where
  * user logs in on day 1 / token expires on day 3 / session expires in 10 weeks
  * if the script runs on  day 5, the session would be deleted according the the original logic